examples/wasm_examples/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "subxtwasm"
```

Up to you whether to call this `wasm-example` or `subxt-web-app-example` or something like that

```suggestion
name = "subxt-web-app-example"
```